gulpfile.js

```diff
@@ -0,0 +1,101 @@
+var _ = require('lodash');
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var ts = require('gulp-typescript');
+var tslint = require('gulp-tslint');
+var realTslint = require('tslint');
+var spawnSync = require('child_process').spawnSync;
+
+function build(src, dest) {
+  var tsProject = ts.createProject('configs/tsconfig-base.json', { declaration: true });
+  return gulp.src(src)
+    .pipe(tsProject())
+    .pipe(gulp.dest(dest));
+}
+
+gulp.task('build:capnp', function () {
+  return build('./packages/capnp-ts/src/**/*.ts', 'packages/capnp-ts/lib');
+});
+
+gulp.task('build:capnpc', function () {
```

Comment:
Call this `build:capnpc-ts` instead, since it looks like we'll be introducing `capnpc-js` as a separate package soon.